spacy/language.py

```diff
@@ -771,8 +771,10 @@ def replace_pipe(
         self.remove_pipe(name)
         if not len(self.pipeline):  # we have no components to insert before/after
             self.add_pipe(factory_name, name=name)
```

Comment:
Since `after=pipe_index - 1` is equivalent to `last=True` (default behaviour), maybe we can combine those two conditions above? Then we can save that extra conditional.